commands/stats.js

```diff
@@ -43,65 +43,70 @@ const stats = msg => {
 			msg.reply("Sorry. This user has missing data. Boo boo doo de doo.");
```

Comment:
Did you want to change this to a message.edit as well for consistency?  Not sure how we'd test it